### PR TITLE
Fixed items from issue 278 and added documentation

### PR DESCRIPTION
**Pull request by lalov1**
I think I got all the items listed in issue #278 and added some documentation.

### CODE REVIEW

File: components/01-visual-styling/02-typography/09-table/table.scss [hunk 1]
```diff
@@ -1,34 +1,58 @@
...
+	&:nth-child(2n) {
+		background-color: $white-b;
+		border-top: 1px solid $grey;
+	}
+}
+
+.table-design thead {
+	background-color: $orange-a11y;
+}
+
```
Comment (garza-utsa) [inline components/01-visual-styling/02-typography/09-table/table.scss:55]: @lalov1 can we also add a couple of more variants here for different color thead options, I'm thinking .table-design-grey, .table-design-blue for this same styling but with a grey or blue thead as variants

**Review by garza-utsa — CHANGES_REQUESTED** [review]
add two more variants for table-design (grey and blue), the grey should be our $grey-d or $grey-c to provide enough contrast from the striped rows

**Comment by prd409-utsa** [comment]
@lalov1  Giving this a bump.  Your changes look good but John David has asked for a few more variants.

File: components/01-visual-styling/02-typography/09-table/table.scss (hunk 1 above)
Comment (lalov1) [reply components/01-visual-styling/02-typography/09-table/table.scss:81]: I've added the new striped variants. Let me know if they should be nested under Striped or if you're okay with how they are located. Also, I used a mixin, but if there is a better way to create the new styles, let me know.

**Comment by lalov1** [comment]
> add two more variants for table-design (grey and blue), the grey should be our $grey-d or $grey-c to provide enough contrast from the striped rows

For the $grey-d the contrast wasn't passing accessibility checks, so I used $grey-c. I did use $grey-d for the border, though.

File: components/01-visual-styling/02-typography/09-table/table.scss (hunk 1 above)
Comment (prd409-utsa) [reply components/01-visual-styling/02-typography/09-table/table.scss:81]: I think that there might not be enough contrast between the colored row and its text, accessability-wise?

Maybe make the text of the colored row white? (Example of CSS)
Screen Shot 2022-06-20 at 4 28 35 PM

Or make the colored row background lighter by using rgba? (Example of CSS)
Screen Shot 2022-06-20 at 4 32 34 PM

File: components/01-visual-styling/02-typography/09-table/table.scss [hunk 2]
```diff
@@ -1,34 +1,58 @@
...
+.table-design th, .table-design td {border: 0px;}
+
+
+
+.table-design tbody>tr {
+	border-bottom: 1px solid $grey;
+
+	&:nth-child(2n) {
```
Comment (prd409-utsa) [inline components/01-visual-styling/02-typography/09-table/table.scss]: I think that there might not be enough contrast between the colored row and its text, accessability-wise?

Maybe make the text of the colored row white? (Example of CSS)
Screen Shot 2022-06-20 at 4 28 35 PM

Or make the colored row background lighter by using rgba? (Example of CSS)
Screen Shot 2022-06-20 at 4 32 34 PM

**Review by prd409-utsa — CHANGES_REQUESTED** [review]

File: components/01-visual-styling/02-typography/09-table/table.scss (hunk 2 above)
Comment (lalov1) [reply components/01-visual-styling/02-typography/09-table/table.scss]: I did check the contrast with the Wave tool, and it seemed to pass with the $grey-c, but not the $grey-d.

**Pull request merged**